File: c/c++/Switch-CaseATM.cpp
#include <iostream>
using namespace std;

 int main(){
    char nama[70];
    bool ismoza = true;

    cout<<"masukan nama : ";
    cin>>nama;

    if (string(nama) !="moza")
    {
        ismoza = false;
        cout<<"masukan lagi nama :"<<endl;
        return main();
    }
    
    int nim;
    cout<<"masukan NIM";
    cin>>nim;

    int saldo = nim;
    cout<<"saldo awal kamu : Rp"<<saldo;

    cout<<"masukan lagi nama :"<<nama<<endl;
    cout<<"saldo awal kamu : Rp"<<saldo<<endl;

    int pilih;
     cout<< "\n=== MENU ATM ==="<< endl;
    cout<< "1. Cek Saldo"<< endl;
    cout<< "2. Tarik Tunai" << endl;
    cout<< "3. Setor Tunai" << endl;
    cout<< "4. Transfer" << endl;
    cout<< "5. Keluar" << endl;
    cout<< "Pilih menu (1-5): ";
    cin>> pilih;

    int menu;
    cout<<"pilih menu";
    cin>>menu;
    switch (menu)
    {
    case 1: cout << "Saldo kamu : Rp" << saldo << endl; break;
    case 2: cout << "tarik tunai (untuk sekarang tidak bisa)"<< endl; break;
    case 3: cout << "setor tunai (untuk sekarang tidak bisa)"<< endl; break;
    case 4: cout << "transfer (untuk sekarang tidak bisa)"<< endl; break;
    case 5: cout << "keluar"<< endl; break;
    
    default: cout<<"tidak ada"<<endl;break;
    }
    return 0;
}

File: php/Switch-CaseATM.php
<?php
function main() {
    $nama = readline("masukan nama : ");
    if ($nama != "moza") {
        echo "masukan lagi nama :" . PHP_EOL;
        return main(); // sama seperti return main() di C++
    }
    $nim = (int) readline("masukan NIM : ");
    $saldo = $nim;
    echo "saldo awal kamu : Rp$saldo\n";
    echo "masukan lagi nama : $nama\n";
    echo "saldo awal kamu : Rp$saldo\n";

    echo "\n=== MENU ATM ===\n";
    echo "1. Cek Saldo\n";
    echo "2. Tarik Tunai\n";
    echo "3. Setor Tunai\n";
    echo "4. Transfer\n";
    echo "5. Keluar\n";
    $pilih = readline("Pilih menu (1-5): ");

    $menu = readline("pilih menu: ");

    switch ($menu) {
        case 1:
            echo "Saldo kamu : Rp$saldo\n";
            break;

        case 2:
            echo "tarik tunai (untuk sekarang tidak bisa)\n";
            break;

        case 3:
            echo "setor tunai (untuk sekarang tidak bisa)\n";
            break;

        case 4:
            echo "transfer (untuk sekarang tidak bisa)\n";
            break;

        case 5:
            echo "keluar\n";
            break;

        default:
            echo "tidak ada\n";
            break;
    }

    return 0;
}

main();
